cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -986,7 +986,13 @@ private Future<MapAndStatus<ConfigMap, KafkaRebalanceStatus>> onReady(Reconcilia
             return Future.succeededFuture();
         }
 
-        LOGGER.debugCr(reconciliation, "KafkaRebalance {} with state [{}] and {}={}",
+        KafkaRebalanceAnnotation rebalanceAnnotation = rebalanceAnnotation(kafkaRebalance);
+        if (rebalanceAnnotation == KafkaRebalanceAnnotation.template) {
+            LOGGER.infoCr(reconciliation, "KafkaRebalance {} is a template configuration. Skipping it.", kafkaRebalance.getMetadata().getName());
+            return Future.succeededFuture();
+        }
```

Comment:
This smells like something that will log again and again every 2 minutes for every template that exists?

Reply:
That's right but I will move this log to TRACE level (I think even DEBUG is too much).